api/src/main/java/org/openmrs/api/impl/PersonServiceImpl.java

```diff
@@ -949,8 +951,9 @@ public RelationshipType retireRelationshipType(RelationshipType type, String ret
 		return Context.getPersonService().saveRelationshipType(type);
 	}
 	
-	/**
+	/**.
 	 * @see org.openmrs.api.PersonService#unretireRelationshipType(org.openmrs.RelationshipType)
+	 *
 	 */
 	@Override
 	public RelationshipType unretireRelationshipType(RelationshipType relationshipType) {
```

Comment:
this file should not show in your commit. remove all changes you made here please.